symfony/plugins/orangehrmPerformancePlugin/entity/PerformanceTrackerLog.php

```diff
@@ -0,0 +1,279 @@
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="ohrm_performance_tracker_log")
+ * @ORM\Entity
+ */
+class PerformanceTrackerLog
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer", length=11)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private int $id;
+
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(name="log", type="string", length=150, nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?string $log;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(name="comment", type="string", length=3000,nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?string $comment;
+
+    /**
+     * @var int|null
+     *
+     * @ORM\Column(name="status", type="integer", length=11, nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?int $status;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(name="achievement", type="string", length=45,nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?string $achievement;
+
+    /**
+     * @var DateTime|null
+     *
+     * @ORM\Column(name="added_date", type="datetime",nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?DateTime $added_date;
+
+    /**
+     * @var DateTime|null
+     *
+     * @ORM\Column(name="modified_date", type="datetime",nullable=true)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private ?DateTime $modified_date;
+
+
+    /**
+     * @var PerformanceTracker|null
+     *
+     * @ORM\ManyToOne (targetEntity="OrangeHRM\Entity\PerformanceTracker", inversedBy="PerformanceTrackerLog")
+     * @ORM\JoinColumns({
+     *   @ORM\JoinColumn(name="performance_track_id", referencedColumnName="id")
+     * })
+     */
+    private ?PerformanceTracker $performanceTracker;
+
+    /**
+     * @var Employee|null
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\Employee", inversedBy="PerformanceTrackerLog")
+     * @ORM\JoinColumns({
+     *   @ORM\JoinColumn(name="reviewer_id", referencedColumnName="emp_number")
+     * })
+     */
+    private ?Employee $employee;
+
+    /**
+     * @var User|null
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\User", inversedBy="PerformanceTrackerLog")
+     * @ORM\JoinColumns({
+     *   @ORM\JoinColumn(name="user_id", referencedColumnName="id")
+     * })
+     */
+    private ?User $systemUser;
```

Comment:
```suggestion
    private ?User $user;
```
:thinking:

Reply:
the **$systemUser** name was in the google  drive document that you shared me